deploy/rosa-console-branding/rosa-branding.console.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed
+  authentication:
+    # the logout redirect must be a valid URI ethat starts with http(s)
+    # valid examples: http://localhost:8080/logout.html, https://www.redhat.com/log/out/
+    logoutRedirect: "https://www.redhat.com/i/should/logout"
+  customization:
+    customProductName: Red Hat OpenShift Service on AWS
```

Comment:
Need confirmation the product name is as expected.  PM question, please raise in jira.

Reply:
Confirmed in JIRA